services/frontend-service/src/legacy-ui/App/index.tsx

```diff
@@ -52,16 +52,37 @@ export const Spinner: React.FC<any> = (props: any) => {
 };
 
 const GetOverview = (props: { children: (r: api.GetOverviewResponse) => JSX.Element }): JSX.Element | null => {
-    const getOverview = React.useCallback(
-        (api: any, authHeader: AuthHeaderType) => api.overviewService().StreamOverview({}, authHeader),
-        []
-    );
-    const overview = useObservable<api.GetOverviewResponse>(getOverview);
+    const [overview, setOverview] = useState<UnaryState<api.GetOverviewResponse>>({ state: 'pending' });
+    const Tdelay = 10 * 1000;
+    const api = React.useContext(Context);
+    const { authHeader } = React.useContext(AuthTokenContext);
+
+    const updateOverview = React.useCallback(() => {
+        api.overviewService()
+            .GetOverview({}, authHeader)
+            .then(
+                (result) => setOverview({ result, state: 'resolved' }),
+                (error) => setOverview({ error, state: 'rejected' })
+            );
+    }, [api, authHeader]);
```

Comment:
Any `useUnary` can be replaced with this callback. 
previously, we had tons of API calls. so a template was useful. now, not so much.